targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/pwmout_api.c

```diff
@@ -124,8 +124,25 @@ void pwmout_period_us(pwmout_t* obj, int us)
     FTM_Type *base = ftm_addrs[obj->pwm_name >> TPM_SHIFT];
     float dc = pwmout_read(obj);
 
-    // Stop FTM clock to ensure instant update of MOD register
-    base->MOD = FTM_MOD_MOD((pwm_clock_mhz * (float)us) - 1);
+    uint32_t pwm_base_clock;
+    uint32_t clkdiv = 0;
+    pwm_base_clock = CLOCK_GetFreq(kCLOCK_BusClk);
+    pwm_clock_mhz = (float)pwm_base_clock / 1000000.0f;
+    uint32_t mod = (pwm_clock_mhz*(float)us) - 1;
+    while(mod > 0xFFFF){
```

Comment:
this is target code but should be same as the rest of the code here, see https://os.mbed.com/docs/latest/reference/style.html

spaces around operators and `while ()`, etc. Please fix